components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -155,11 +157,15 @@ const SearchView = () => {
 
         Meteor.subscribe(Meteor.settings.public.AggregationResultsCollectionName, {
             aggregationJobId: aggregationJobId,
+            isExpectingUpdates: SEARCH_SIGNAL.REQ_QUERYING === resultsMetadata.lastSignal,
```

Comment:
```suggestion
            isExpectingUpdates: [
                SEARCH_SIGNAL.REQ_QUERYING,
                SEARCH_SIGNAL.RESP_QUERYING
            ].includes(resultsMetadata.lastSignal),
```

Reply:
Consider de-duplicating the code. e.g., Insert this before `    const searchResults = useTracker(() => {`:

```
    const isExpectingUpdates = (null !== searchJobId) && [
        SEARCH_SIGNAL.REQ_QUERYING,
        SEARCH_SIGNAL.RESP_QUERYING
    ].includes(resultsMetadata.lastSignal);
```

We can compute `isExpectingUpdates` outside of the useTracker callbacks so that the result can be used by both subscriptions.

Reply:
If moving `isExpectingUpdates` outside, should we make it reactive? i.e. set up a `useEffect` to track `lastSignal`

Reply:
Good catch. You can look into [useMemo](https://react.dev/reference/react/useMemo).